docs/maintainers/determining-concepts.md

```diff
@@ -14,11 +14,34 @@ When starting this process, we **strongly** recommend you seek help from others
 
 ## Concept Exercises
 
-Generally, only **one new Concept** should be introduced per Concept Exercise. 
+Generally, only **one new Concept** should be introduced per Concept Exercise.
 
 In order to keep the exercises interesting, maintainers may choose to introduce multiple _trivial_ Concepts in one exercise.
 A _trivial_ Concept is one where we can reasonably assume that most programmers will quickly grasp it, due to prior familiarity, albeit with differing syntax. Examples might be basic number usage (`+`, `-`, `*`, etc.), or conditionals (`if/else`).
 
 Every new Concept introduced, regardless of how _trivial_ it may seem, must be explicitly mentioned in the exercise's introduction. Each mention should focus on how that Concept is applied in the given language, with links to learn more on the topic. If there is nothing special about it, a language-agnostic link is helpful.
 
 All Concepts covered in a Concept Exercise must either be taught by it, or must be marked as a prerequisite in the `config.json`.
+
+### Naming Concepts
+
+Concept Exercise concepts (as used in the `config.json` file) should be named according to the following rules:
+
+1. Use lowercase.
+1. Use [kebab-case][kebab-case].
+1. Don't use a hierarchy.
+
+Good examples:
+
+- `classes`
+- `bit-manipulation`
+- `floating-point-numbers`
+
+Bad examples:
+
+- `INHERITANCE`: should use lowercase (`inheritance`)
+- `AnonymousFunctions`: should use use kebab-case (`anonymous-functions`)
+- `methods.parameters.optional`: should not use a hierarchical notation (`optional-parameters`)
+- `variables-basic`: should not use a hierarchical notation (`variables`)
```

Comment:
Suggest changing this to `numbers-floating-point` and also added `numbers/floating-point` as another example. The reuse of the good examples helps clarify I think.

Reply:
I had that originally, but I felt that was maybe hammering home the point to bluntly. But is does help clarify, so I'll update.

Reply:
Updated